src/OrbitGl/App.cpp

```diff
@@ -1146,7 +1152,28 @@ void OrbitApp::StartCapture() {
       data_manager_->max_local_marker_depth_per_command_buffer();
 
   bool collect_memory_info = data_manager_->collect_memory_info();
-  uint64_t memory_sampling_period_ns = data_manager_->memory_sampling_period_ns();
+  uint64_t memory_sampling_period_ms = data_manager_->memory_sampling_period_ms();
+
+  std::optional<int64_t> memory_information_sampling_period_ms_opt = std::nullopt;
+  if (collect_memory_info) {
+    memory_information_sampling_period_ms_opt = static_cast<int64_t>(memory_sampling_period_ms);
+  }
+
+  bool lib_orbit_vulkan_layer_loaded = false;
+  std::vector<const ModuleData*> vulkan_layer_modules =
+      module_manager_->GetModulesByFilename(kLibOrbitVulkanLayerSoFileName);
+  for (const auto& module : vulkan_layer_modules) {
```

Comment:
This is more or less referring to the question above. We do not require the user to load the Vulkan layer. Only the game needs to load it. If this is the case here, all fine, otherwise, we are doing too much work.

Anyways, could you add a brief comment here with out assumption, that the layer is activated if the game/target has it in its modules list?

Reply:
I added a comment and I renamed `lib_orbit_vulkan_layer_loaded` to `orbit_vulkan_layer_loaded_by_process` because I think its more readable.